ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/holesky.yaml

```diff
@@ -41,6 +41,10 @@ DENEB_FORK_EPOCH: 29696
 ELECTRA_FORK_VERSION: 0x06017000
 ELECTRA_FORK_EPOCH: 115968
 
+# Fulu
+FULU_FORK_VERSION: 0x07017000
+FULU_FORK_EPOCH: 18446744073709551615
+
```

Comment:
we should actually be able to put all the fulu fields in and be standard to the testconfig again iirc...